js/components/portal-dashboard/question-area.tsx

```diff
@@ -26,6 +26,8 @@ export class QuestionArea extends React.PureComponent<IProps>{
     const questionType = QuestionTypes.find(qt => qt.type === type && qt.scored === scored);
     const QuestionIcon = questionType?.icon;
     const activityURL = currentActivity?.get("url");
+    const page = (currentQuestion?.get("page").split("_"))[1];
```

Comment:
I notice when I run the test sequence that page is undefined because in our sample data the page and page number are separated by "-" not "_".  Which is correct?  Do we need to test for undefined regardless?  Without a test, it tries to open:
http://app.lara.docker/activities/9/pages/undefined/